pkg/infra/tracing/test_helper.go

```diff
@@ -0,0 +1,16 @@
+package tracing
+
+func InitializeTracerForTest() (TracerService, error) {
+	ots := &Opentelemetry{}
+	err := ots.initOpentelemetryTracer()
+	if err != nil {
+		return ots, err
+	}
+	return ots, err
+}
+
+func InitializeForBus() TracerService {
```

Comment:
Nit. This one should probably live in the bus package. Does this mean that the usage of global bus would always use the Opentelemetry tracer even though OpenTracing has been enabled? Probably need to fix that in that case, either by exposing a global tracer or removing the use of the global bus.

Reply:
the function that is called from InitializeForBus is unexported in tracing pkg